ui/src/components/SchemaTable/index.jsx

```diff
@@ -1,15 +1,22 @@
 import React, { Component } from 'react';
 import { withRouter } from 'react-router-dom';
-import { join } from 'path';
 import RefParser from 'json-schema-ref-parser';
 import { string } from 'prop-types';
 import Spinner from '@mozilla-frontend-infra/components/Spinner';
 import { withStyles } from '@material-ui/core/styles';
 import Table from 'react-schema-viewer/lib/SchemaTable';
+import cloneDeep from 'lodash.clonedeep';
+import jsonSchemaDraft06 from 'ajv/lib/refs/json-schema-draft-06.json';
+import jsonSchemaDraft07 from 'ajv/lib/refs/json-schema-draft-07.json';
 import ErrorPanel from '../ErrorPanel';
 import { THEME } from '../../utils/constants';
 import references from '../../../../generated/references.json';
 
+// Local copies of the json-schemas schemas, since TC schemas $refer to these
+const EXTERNAL_SCHEMAS = [jsonSchemaDraft06, jsonSchemaDraft07].reduce(
+  (schemas, schema) => ({ ...schemas, [schema.$id]: schema })
+);
+
```

Comment:
You'll probably want to add `{}` as the initial value otherwise it won't work.

Reply:
It does, surprisingly!  I guess `{...undefined}` is equivalent to `{}`?  But I'll add the `{}`.